accounts/templates/accounts/login.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'events/main.html' %}
+{% block content %}
+    <div class="container h-100">
+        <div class="d-flex justify-content-center h-100">
+            <div class="user_card">
+                <div class="d-flex justify-content-center">
+                    <h3 id="form-title" class="my-5">LOGIN</h3>
+                </div>
+                <div class="d-flex justify-content-center form_container">
+
+                    <form class="form-group" method="POST" action="">
+                        {% csrf_token %}
+                        <div class="input-group mb-3">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text">Username</span>
+                            </div>
+                            <input type="text" name="username" class="form-control"/>
+                        </div>
+                        <div class="input-group mb-3">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text">Password</span>
+                            </div>
+                            <input type="password" name="password" class="form-control"/>
+                        </div>
+
+                        <div class="d-flex justify-content-center mt-3 login_container">
+                            <input class="btn login_btn btn-success my-4" type="submit" value="Login">
+                        </div>
+                        <a href="{% url "social:begin" "google-oauth2" %}">
+                            <div class='text-center'>
```

Comment:
You should make the div with the 'text-center' class to wrap all of the icon and text, so you won't need to add a 'text-center' class to every div.

Reply:
ok fixed it!